ui/src/logs/containers/LogsPage.tsx

```diff
@@ -347,7 +351,81 @@ class LogsPage extends Component<Props, State> {
             colors={histogramColors}
             onBarClick={this.handleBarClick}
             sortBarGroups={this.handleSortHistogramBarGroups}
-          />
+          >
+            {({adjustedWidth, adjustedHeight, margins}) => {
```

Comment:
What do you think of making this into its own component? Not a blocker for merging, but might be nicer to read